CHANGELOG.md

```diff
@@ -736,7 +736,7 @@ Big thanks to community contributor @Akronix!
 
 ## [0.14.0] - 2017-10-17
 ### :sparkles: Added
-- An `Upload` component! :tada: See [https://plot.ly/dash/dash-core-components/upload](https://plot.ly/dash/dash-core-components/upload) for docs.
+- An `Upload` component! :tada: See [https://plotly.com/dash/dash-core-components/upload](https://plotly.com/dash/dash-core-components/upload) for docs.
```

Comment:
Should I change this to dash.plotly.com/dash/... instead?

Reply:
Yes. Or rather `dash.plotly.com/dash-core-components`

Reply:
No. Redirects are handling these just fine.